_includes/post_footer.html

```diff
@@ -0,0 +1,3 @@
+<div class="post-footer">
+  <a href="/articles/">View all articles</a>
```

Comment:
sorry, I'm deleting the navigation file, we aren't using it, so we don't really have a standard to be consistent.

Which do you prefer? with or without  {{ site.baseurl }} ?